protocol/transport/whisper/whisper_service.go

```diff
@@ -360,6 +361,10 @@ func (a *WhisperServiceTransport) Track(identifiers [][]byte, hash []byte, newMe
 	}
 }
 
+func (a *WhisperServiceTransport) GetCurrentTime() uint64 {
```

Comment:
Let's add a comment or a unit test that makes sure it returns unix timestamp in milliseconds.